docs/contributing.rst

```diff
@@ -43,12 +43,16 @@ To make a release and **deploy to PyPI**, please follow these steps (we highly s
 
 The flow would then look like follows:
 
-1. :code:`bumpversion (patch|minor)`
-2. :code:`make docs`
-3. :code:`git add CHANGELOG.*`
-4. :code:`git commit -m "update changelog"`
-5. :code:`git push`
-6. :code:`git push --tags`
+1. :code:`bumpversion patch` or :code:`bumpversion minor`
+2. :code:`git describe --tags`, and note this tag name. We will need to edit this tag later.
```

Comment:
would you here expect a tag with the version you're trying to release?

Reply:
Yes